src/cljs/rems/administration/create_form.cljs

```diff
@@ -140,7 +140,10 @@
           :field/type (:field/type field)
           :field/optional (if (common-form/supports-optional? field)
                             (boolean (:field/optional field))
-                            false)}
+                            false)
+          }
+         (when (:field/info-text field)
```

Comment:
@opqdonut actually .. it is here ...

Reply:
heh indeed, well my comments still stand :)

Reply:
😸